src/main/groovy/br/com/bluesoft/bee/database/reader/PostgresDatabaseReader.groovy

```diff
@@ -71,6 +71,7 @@ class PostgresDatabaseReader implements DatabaseReader {
         } catch (Exception e) {
             databaseVersion = sql.rows("select setting from pg_settings where name = 'server_version'")[0].get('setting')
         }
+        databaseVersion = databaseVersion.split(" ")[0]
```

Comment:
this code probably breaks VersionHelper.isNewerThan9_6

--

esse código provavelmente quebra o VersionHelper.isNewerThan9_6